### DISCUSSION TITLE
How to create a table of contents on the side?

### DISCUSSION THREAD

**Discussion by TECH-JEFF** [requarks/wiki#3485] [Help / Questions]
**Question**
How to create a table of contents on the side?

**Host Info (please complete the following information):**
OS: Server 2016
Wiki.js version: 2.5.126
Database engine: Postgres 12

**Comment by Smankusors** [comment]
use headings

---

btw what editor do you use?

**Comment by TECH-JEFF** [comment]
> use headings
> 
> btw what editor do you use?

I used the default visual editor
